frontend/src/js/controllers/challengeCreate.js

```diff
@@ -9,7 +9,7 @@
 
     ChallengeCreateCtrl.$inject = ['utilities', '$state', '$http', '$rootScope'];
```

Comment:
@spyshiv The eslint wasn't showing any error thats why i forgot to remove these injections.
Done with this.